api/rest-api-filters/hook.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+$filters = get_option( WCA_Test_Helper_Rest_Api_Filters::WC_ADMIN_TEST_HELPER_REST_API_FILTER_OPTION );
+
+function array_dot_set( &$array, $key, $value ) {
+    if ( is_null( $key ) ) {
+        return $array = $value;
+    }
+
+    $keys = explode( '.', $key );
+
+    while ( count( $keys ) > 1 ) {
+        $key = array_shift( $keys );
+        if (! isset( $array[$key] ) || ! is_array( $array[$key]) ) {
+            $array[$key] = [];
+        }
+        $array = &$array[$key];
+    }
+
+    $array[ array_shift($keys) ] = $value;
+    return $array;
+}
+
+add_filter(
+    'rest_request_after_callbacks',
+    function( $response, array $handler, \WP_REST_Request $request ) use ( $filters ) {
+		if ( ! $response instanceof  \WP_REST_Response ) {
+			return $response;
+		}
+	    $route = $request->get_route();
+		$filters = array_filter( $filters, function( $filter ) use ( $request, $route ) {
```

Comment:
@moon0326 Looks like our PHP code isn't well-formatted. Do we need to install [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer)?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin-test-helper/pull/37/commits/c3ad996eb28adba93a51fb9a5bc0a37925eee94f.

I'll work a follow up PR to add PHP linters